src/Loader.php

```diff
@@ -325,9 +325,9 @@ protected function beginsWithAQuote($value)
     public function getEnvironmentVariable($name)
     {
         switch (true) {
-            case array_key_exists($name, $_ENV):
+            case isset($_ENV[$name]):
```

Comment:
@rquadling The only value that `isset()` does not like is `null`. Other falsy values, such as `false`, `0`, `''` (the empty string) and `'0'` (the string 0) are considered to be "set". On the other hand, `empty()` and implicit/automatic boolean conversion do not like those values: https://secure.php.net/manual/en/language.types.boolean.php